src/types/mod.rs

```diff
@@ -80,6 +80,20 @@ impl StacksNetwork {
             _ => false,
         }
     }
+
+    pub fn either_devnet_or_tesnet(&self) -> bool {
+        match self {
+            StacksNetwork::Simnet | StacksNetwork::Devnet => true,
```

Comment:
Shouldn't this be `Testnet`, not `Simnet`? Or should the function be renamed?

Reply:
Addressed in https://github.com/hirosystems/clarinet/pull/388/commits/895ba873a7a36f7218ce35e4f9e081fb5d4d18fb